File: client/src/components/Author.js
import {
  ListGroup,
  ListGroupItem,
  ListGroupItemHeading,
  
} from "reactstrap";

const Author = (props) => {
  const { authorId, authorName, authorLastName } = props.author;

  return (
    <div>
      <ListGroup>
        <ListGroupItem>
          <ListGroupItemHeading>
            {authorName} {authorLastName}
          </ListGroupItemHeading>
        </ListGroupItem>
      </ListGroup>
    </div>
  );
};

export default Author;

File: BookstoreBackend/Bookstore.Entities/Author.cs
using Bookstore.Core.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bookstore.Entities
{
    public class Author:IEntity
    {
        public int AuthorId { get; set; }
        public string AuthorName { get; set; }
        public string AuthorLastName { get; set; }

    }
}
